terraform/dev/main.tf

```diff
@@ -87,6 +87,21 @@ resource "cloudfoundry_service_instance" "redis" {
   recursive_delete = true
 }
 
+###
```

Comment:
We will need this added to staging before we merge (and prod before we merge to prod). Do you want to pair getting it to staging?

Reply:
Lets pair this afternoon. Going to add a testing path to API.